### PR TITLE
Independent test loader

### PR DESCRIPTION
**Pull request by rudovjan**
Please take a look and give a feedback on approach 👍 

This enables integration with cmd line and with ci and with IDEs

Notes:
1. wonder if there is a real need to have a running application for unit tests...

Verified with demo app using this branch and with test

```
node app.js test
```

is same as

```
/node_modules/mocha/bin/_mocha /node_modules/quadro/lib/test/jest_config.js --ui tdd test/**/*_test.js
```

```js
describe('Test of globals', function() {
	it('globals', () => {
		expect(global.pubsub).not.to.equal(undefined);
		expect(QT).not.to.equal(undefined);
		expect(QT.chai).not.to.equal(undefined);
		expect(Q).not.to.equal(undefined);
		expect(QT.stubConfig).not.to.equal(undefined);
		expect(QT.currentTest).not.to.equal(undefined);
		expect(QT.onMessage).not.to.equal(undefined);
		expect(QT.httpTest).not.to.equal(undefined);
		expect(QT.request).not.to.equal(undefined);
	});
});
```

### CODE REVIEW

**Comment by rudovjan** [comment]
That's for first time ever in my JS carrier where PR failes on "Extra semicolon." 🗡 🍡 

What's wrong with semicolons? 🔨

**Comment by rudovjan** [comment]
This could help to jest too.

File: lib/test/test.js [hunk 1]
```diff
@@ -36,7 +28,6 @@ module.exports = class {
   }
 
   async run() {
-    global.container = this.container
```
Comment (rudovjan) [inline lib/test/test.js]: This can be saved.. but I hope noone is using global.container but just Q.container

File: lib/test/initialize_tests.js [hunk 2]
```diff
@@ -7,7 +7,15 @@ async function runInitializers(app, container, log) {
 }
 
 before(async function() {
-  let container = global.container
```
Comment (rudovjan) [inline lib/test/initialize_tests.js]: I assume using global.container was not intentional.

File: lib/test/test.js (hunk 1 above)
Comment (igorshapiro) [reply lib/test/test.js]: +1 Initially a couple of key entities was exposed via global (like `describe` in mocha). But later we've wrapped it all under `Q` (quadro) and `QT` (quadro test) namespaces

**Comment by rudovjan** [comment]
> Need also to update CHANGELOG with the next version

Make sense. Plus would be cool if someone could try that with any existing service & tests @ondrej-kvasnovsky @darthburrito @ankurjain86

**Comment by rudovjan** [comment]
@igorshapiro bumped version + changed change log

**Pull request closed without merging**